src/test/resources/safeRemovedFields.txt

```diff
@@ -0,0 +1,5 @@
+#Contains a list of fields that were *safely* removed. Safe removal involves doing an upgrade test to see that
+#the class can be removed without breaking any jobs or other functionality.
+#Very few fields should be listed in here, and once a release is made the field should be removed from here (or the test fails)
+#Format is semi-colon separated string, with fully-justified field name
+#fullClassName.fieldName;reason why it is safe to remove
```

Comment:
why not the more standard `fullClassName#fieldName`? I am slightly (but not very) concerned that you have clashes here between internal classes and fields, it is very very minor, but why not opt for the more standard "#" to separate the class and the field?

Reply:
For posterify- I was following the toString convention on the Field class. `className#fieldName` is javadoc convention which I agree is better here for disambiguating packages/inner classes etc.